### ISSUE TITLE
How to request edit access?

### ISSUE THREAD

**Issue by foolip** [GoogleChrome/chromium-dashboard#1144]
In https://chromium-review.googlesource.com/c/chromium/src/+/2578703 I requested that @Honry add a chromestatus.com entry for `setTimeout` clamping. However, I can't see how to request edit access. There used to be a link, but it was removed in 04afe6964cdf8f54fac7e33753e423920ae3c49e and not replaced by anything else.

The [FAQ](https://docs.google.com/document/d/1jrSlM4Yhae7XCJ8BuasWx71CvDEMMbSKbXwx7hoh1Co/edit?pli=1) mentions edit-bug-access@chromium.org, but "edit-bug-access" doesn't appear in the source code, so I doubt that's correct.

**Comment by foolip** [comment]
This also came up [Intent to ship: Percent based scrolling](https://groups.google.com/a/chromium.org/g/blink-dev/c/5Mt8RZyf-pc/m/up-c0DPUCgAJ) where @rahul8805 was unable to edit an existing entry.

**Comment by GrumpyOldTroll** [comment]
I ran into this and addressed it slightly differently by sending an email to blink-api-owners-discuss@chromium.org.

It asks for an update to the "launching-features" doc page, whereas I think this issue is asking to change the chromestatus/features page to give something like a link to request access, and on reflection maybe that's a better place for it (or both).

But they asked to add this message to the bug report, so here's what I sent (to which they responded by giving me access, and seem to be taking further action to fix it better for the future):

-----

Hi Blink-api-owners,

I'm writing about the process documented here:
https://www.chromium.org/blink/launching-features

We're trying to launch a new feature, and we got stuck for a bit on the
"Create a Chromestatus entry" step.

I'd like to make a suggestion for the doc:

- it would be helpful to add a point of contact for gaining permission
  to create a Chromestatus entry, for those who don't have it.

I'd be happy to describe the saga in more detail if you'd like more
details to do a post-mortem, but I'll spare you unless you ask for it.

The tl;dr is that I believe this hopefully-simple change could have
prevented a fair amount of wheel-spinning and noise, both for us as
aspiring new contributors and for the reviewers who were kind enough
to take a look at our early attempts at getting some motion.

(There were a few different contributing layers of confusion, and in
retrospect I can also think of some other things I could have done
differently that might have made for a better result, but having this
info in the doc would have been a big help, and hopefully this can
make a smoother path for the next people who hit this.)

Thanks and regards,
Jake

PS: the doc recommends making suggestions like this via "feedback" on
the google doc version of the page, with a link here:
https://docs.google.com/document/d/1R_2V5_rukVM8S5Hg2XKxiEOVRxvB7NqZOI2lfni2YwE

However, I also don't have permissions to comment on that page, so
I'm writing here instead.  This also could probably use tweaking, but
is less of a problem since a group email is at least provided.

**Comment by mrego** [comment]
Yeah, some igalians get confused every now and then about this. Though some of us have permissions and can add/modify the entries for them.

If I'm not wrong, all @chromium.org accounts have edit permissions. But people that are not committers yet, don't have a @chromium.org account. They're in a status in which they can land a patch in https://chromium-review.googlesource.com/ that has been reviewed, but they cannot add/edit a chromestatus entry.

**Comment by foolip** [comment]
@jpmedley tells me that one can contact him to get an account registered, but it really would be good to document the process, even if it currently is poking Joe.

**Comment by jrobbins** [comment]
For now, the official process is to send an email to webstatus-request@google.com to request access.
That is now documented on
https://www.chromium.org/blink/launching-features
You would be reaching more or less the same people, it just that we can add or remove group admins as needed.

**Comment by Honry** [comment]
I have the access right now! Thank you all for the support!

**Comment by foolip** [comment]
That's great, thanks @jrobbins! Could it be documented directly on chromestatus.com too?

**Comment by arskama** [comment]
@jrobbins 
I have the same issue as @Honry.
I was really pleased to see that there was a potential solution to our problem and that I could request edit access via email request.
I have sent 2 emails, one in the beginning of Feb and the other one yesterday. I still not have any edit access, though I do have a valid entry in chromestatus, which I need to update.

https://chromestatus.com/features/5384099747332096

**Comment by jpmedley** [comment]
I apologize for failing to notice the request. You've now have access.
Joe Medley | Technical Writer, Chrome DevRel | jmedley@google.com |
 816-678-7195
*If an API's not documented it doesn't exist.*

**Comment by arskama** [comment]
@jpmedley, great! Thanks for granting the access.

**Comment by jpmedley** [comment]
I really apologize. It shouldn't happen for your coworkers. We rolled a new
system for requesting last month and I forgot to set up an inbox filter to
catch them.
Joe Medley | Technical Writer, Chrome DevRel | jmedley@google.com |
 816-678-7195
*If an API's not documented it doesn't exist.*

**Comment by DanielRyanSmith** [comment]
Does any action need to be taken here? I see that the request for access is still documented in the launching features documentation per [@jrobbins' comment](https://github.com/GoogleChrome/chromium-dashboard/issues/1144#issuecomment-771915561). Would it be useful to surface this directly in Chromestatus? Or can we close this issue?
